src/LeftAdminExtension.php

```diff
@@ -4,27 +4,60 @@
 
 use SilverStripe\Control\Director;
 use SilverStripe\Core\Extension;
-use SilverStripe\Security\Group;
-use SilverStripe\Security\Member;
 use SilverStripe\Security\Permission;
-use SilverStripe\Security\Security;
 use SilverStripe\View\Requirements;
 
 class LeftAdminExtension extends Extension
 {
     public function init()
     {
+        // get the UA token and abort if its not configured yet
         $ua = $this->getOwner()->config()->get('primary_ua');
-        if ($ua) {
-            Requirements::customScript( <<<JS
+        if (!$ua) {
+            return;
+        }
+
+        // window.ss
+        Requirements::customScript(<<<JS
 window.ss = window.ss || {};
 window.ss.analytics = {$this->getAnalyticsConfig()};
 JS
-            );
-            Requirements::javascript('https://www.googletagmanager.com/gtag/js?id=' . $ua);
-            Requirements::javascript('maxime-rainville/silverstripe-cms-analytics:client/dist/js/bundle.js');
-        }
+        );
+
+        // We initialise Google Analytics in the header of the page so it tracks load times correctly
+
+        // gtag
+        Requirements::javascript('https://www.googletagmanager.com/gtag/js?id=' . $ua);
+        Requirements::customScript(<<<JS
```

Comment:
That's a lot of custom script that gets loaded in via PHP logic. I would suggest taking the `$ua` variable and injecting as a global JS variable. Then loading everything else via a JS file.

Does that make sense?